sdk/kms/package.json

```diff
@@ -0,0 +1,59 @@
+{
+	"name": "@mysten/kms",
+	"version": "0.0.1",
+	"description": "A collection of KMS signers for various cloud providers",
+	"license": "Apache-2.0",
+	"author": "Mysten Labs <build@mystenlabs.com>",
+	"type": "commonjs",
+	"exports": {
+		"./aws": {
+			"import": "./dist/esm/aws/index.js",
+			"require": "./dist/cjs/aws/index.js"
+		}
+	},
+	"sideEffects": false,
+	"files": [
+		"CHANGELOG.md",
+		"LICENSE",
+		"README.md",
+		"aws",
+		"dist",
+		"src"
+	],
+	"scripts": {
+		"clean": "rm -rf tsconfig.tsbuildinfo ./dist",
+		"build": "build-package",
+		"prepublishOnly": "pnpm build",
+		"prettier:check": "prettier -c --ignore-unknown .",
+		"prettier:fix": "prettier -w --ignore-unknown .",
+		"eslint:check": "eslint --max-warnings=0 .",
+		"eslint:fix": "pnpm run eslint:check --fix",
+		"lint": "pnpm run eslint:check && pnpm run prettier:check",
+		"lint:fix": "pnpm run eslint:fix && pnpm run prettier:fix",
+		"test": "vitest"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/mystenlabs/sui.git"
+	},
+	"bugs": {
+		"url": "https://github.com/mystenlabs/sui/issues"
+	},
+	"homepage": "https://github.com/mystenlabs/sui#readme",
+	"devDependencies": {
+		"@mysten/build-scripts": "workspace:*",
+		"@types/node": "^20.14.10",
+		"typescript": "^5.5.3",
+		"vitest": "^2.0.1"
+	},
+	"dependencies": {
+		"@mysten/sui": "workspace:*",
+		"@noble/curves": "^1.4.2",
+		"@noble/hashes": "^1.4.0",
+		"asn1-ts": "^8.0.2",
+		"aws4fetch": "^1.0.20"
+	},
+	"engines": {
+		"node": ">=20"
```

Comment:
Was this added because one of the deps depends on this or jsut as a best practice?

Reply:
It was added for the dependency needs (access to `crypto` from aws4fetch)

Reply:
You just need the `--experimental-global-webcrypto` flag for node 18.

Reply:
Although to be clear I'm also fine bumping to 20 (honestly I'd also be fine going to 22 given it's the active LTS but idk how agressive we want to be).